src/main/java/com/gd/clinic/security/entity/User.java

```diff
@@ -18,12 +19,13 @@
 @RequiredArgsConstructor
 @NoArgsConstructor
 @Accessors(chain = true)
-@Table(name = "users")
+@Table(name = "Users")
```

Comment:
Yes, it had some issues because int conflicted with another table, changed it back to lowercase and the issue is not occuring anymore. The table name is now lowercase this commit.